Chrysalis-Papilio/Soen-490-Papilio-Android
# [PAPILIO-16] Favorite Activities - Additions

# General info

**Issue number**: PAPILIO-16

**Task description**:  

 - Ability to favorite/unfavorite activities
 - Ability to view favorite activities on a dedicated page
 - General UI improvements

# Testing
To test this feature you have to:

1. Open the app and log-in/sign-up
2. On the activity feed page, tap on any activity to go to its info page
3. On the info page, tap on the heart button to favorite an activity
4. Go to the "My Activities" tab and swipe left to open the "Favorites" tab
5. Verify that the activity you favorited shows up here after loading
6. Tap on that activity again and unfavorite it. Then go back to the "Favourites" page
7. Verify that the activity disappears after a short time. 
8. If steps 5 and 7 are verified successfully. This story is complete.
